internal/manager/run.go

```diff
@@ -171,6 +173,21 @@ func Run(ctx context.Context, c *Config, diagnostic util.ConfigDumpDiagnostic, d
 		return fmt.Errorf("unable to setup readyz: %w", err)
 	}
 
+	if c.Konnect.ConfigSynchronizationEnabled {
+		setupLog.Info("Start Konnect client to register runtime instances to Konnect")
+		konnectClient, err := konnect.NewAdminClient(c.Konnect)
+		if err != nil {
+			setupLog.Error(err, "failed to create konnect client")
+			return fmt.Errorf("failed to create konnect client: %w", err)
+		}
+		hostname, _ := os.Hostname()
+		version := metadata.Release
+		// REVIEW: here we used version of KIC itself as version of KIC node.'
+		// Which version is the proper one to use?
```

Comment:
I believe we should use the KIC version for the KIC node, while for the data-plane nodes we should use their own.